src/microbe_stage/Spawners.cs

```diff
@@ -103,8 +103,8 @@ public static void GiveFullyGrownChanceForMulticellular(Microbe microbe, Random
     // TODO: this is likely a huge cause of lag. Would be nice to be able
     // to spawn these so that only one per tick is spawned.
     public static IEnumerable<Microbe> SpawnBacteriaColony(Species species, Vector3 location,
-        Node worldRoot, PackedScene microbeScene, CompoundCloudSystem cloudSystem,
-        GameProperties currentGame, Random random)
+        Vector3 playerPosition, Node worldRoot, PackedScene microbeScene,
+        CompoundCloudSystem cloudSystem, GameProperties currentGame, Random random)
     {
         var curSpawn = new Vector3(random.Next(1, 8), 0, random.Next(1, 8));
```

Comment:
I know this isn't your code, but maybe it could help to rename this variable to something more explicit (does 'cur' stand for 'current'?)

Reply:
Yeah, it stands for 'current'. Would currentSpawn be good or something else?